airflow/models/dag.py

```diff
@@ -3204,6 +3204,13 @@ def __init__(self, concurrency=None, **kwargs):
     def __repr__(self):
         return f"<DAG: {self.dag_id}>"
 
+    @property
+    def schedule_description(self) -> str | None:
```

Comment:
Type is wrong here I believe:
```suggestion
    def schedule_description(self) -> str:
```